service/src/main/resources/application-user-mappings.yaml

```diff
@@ -0,0 +1,6 @@
+spring:
```

Comment:
The problem I see with this file is that after we start tracking it in repository anyone who generated their own local mapping will see that file as modified and some day someone will push it.

I suggest that:
1. script `get-user-ids.sh` (renamed to `generate-local-user-mappings.sh`) is reimplemented to generate `application-user-mappings.yaml` from scratch
2. file `application-user-mappings.yaml` is added to `.git-ignore` so that no-one will be able to commit and push it

Reply:
cool sounds good & makes sense :)